src/App.test.js

```diff
@@ -1,8 +1,11 @@
-import { render, screen } from '@testing-library/react';
+import { render } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+// Testeamos el logo mediante la propiedad atributo alt
+
+describe('Ecommerce Store logo/name appears in the Header', () => {
+  it('Logo', () => {
+    const {getByAltText} = render(<App/>)
+    getByAltText("shoppingCarIcon");
```

Comment:
You are missing the assertion, could be something like this:
`const carIcon = getByAltText("shoppingCarIcon");
expect(carIcon).toBeInTheDocument();
`
You can take a look at this: https://jestjs.io/docs/expect